configure.py

```diff
@@ -0,0 +1,63 @@
+import os, platform, subprocess
+
+def checkCommand(shell):
+    if platform.system() == "Windows":
+        return os.system("where " + shell)
+    else:
+        return os.system("which " + shell)
+
+def checkAll():
+    print("Checking for valid bash")
+    if checkCommand("bash"):
```

Comment:
Modern macOS systems don't install bash.  They might still have bash in path, aliased to something else, not sure, untested by me.

Reply:
I think I saw a few bash scripts somewhere in the build system

Reply:
I'm not aware of any - did it error when you didn't have bash installed?

Reply:
Not sure how I can temporarily remove bash without bricking my os.

Reply:
In that case I'd say remove the check - we don't have any evidence it's required.  If we find someone hits this problem we can document the circumstances and add it back (or fix the unexpected bash dependency).

Reply:
I can check for `sh` instead, sound good?

Reply:
I don't know.  I think if we don't know what shell we may or may not require, how can we know if we're running a good test?

Probably better to not test this at all, until we see an error from someone and can identify incorrect shell as being the cause.

Reply:
Resolving: check for bash was removed